package.nls.hindi.json

```
@@ -0,0 +1,31 @@
+{
+    "python.command.python.sortImports.title": "आयात क्रमबद्ध करें",
+    "python.command.python.startREPL.title": "REPL शुरू करें",
+    "python.command.python.createTerminal.title": "टर्मिनल बनाएं",
+    "python.command.python.buildWorkspaceSymbols.title": "वर्कस्पेस सिंबल बनाएं",
+    "python.command.python.runtests.title": "सभी टेस्ट चलाएं",
+    "python.command.python.debugtests.title": "सभी टेस्ट डीबग करें",
+    "python.command.python.execInTerminal.title": "टर्मिनल में पायथन फाइल चलाएं",
+    "python.command.python.setInterpreter.title": "इंटरप्रेटर चुनें",
+    "python.command.python.refactorExtractVariable.title": "Variable निकालें",
+    "python.command.python.refactorExtractMethod.title": "Method निकालें",
+    "python.command.python.viewTestOutput.title": "टेस्ट आउटपुट दिखाएँ",
+    "python.command.python.selectAndRunTestMethod.title": "टेस्ट विधि चलाएँ ...",
+    "python.command.python.selectAndDebugTestMethod.title": "डिबग टेस्ट विधि ...",
+    "python.command.python.selectAndRunTestFile.title": "टेस्ट फ़ाइल चलाएँ ...",
+    "python.command.python.runCurrentTestFile.title": "वर्तमान परीक्षण फ़ाइल चलाएँ",
+    "python.command.python.runFailedTests.title": "रन विफल परीक्षण",
+    "python.command.python.execSelectionInTerminal.title": "Eायथन टर्मिनल में चयन या लाइन चलाएं",
+    "python.command.python.execSelectionInDjangoShell.title": "Django शेल में चलाएं या लाइन चलाएं",
+    "python.command.python.goToPythonObject.title": "पायथन ऑब्जेक्ट पर जाएं",
+    "python.command.python.setLinter.title": "लिंटर चुनें",
+    "python.command.python.enableLinting.title": "Linting सक्षम करें",
+    "python.command.python.runLinting.title": "Linting चलाएं",
+    "python.snippet.launch.standard.label": "पायथन: वर्तमान फ़ाइल",
+    "python.snippet.launch.module.label": "पायथन: मॉड्यूल",
+    "python.snippet.launch.django.label": "Python: Django",
+    "python.snippet.launch.flask.label": "Python: Flask",
+    "python.snippet.launch.pyramid.label": "पायथन: पिरामिड अनुप्रयोग",
```

```suggestion
    "python.snippet.launch.pyramid.label": "पायथन: Pyramid अनुप्रयोग",
```
Since we are suing Django/flask etc in english.